### DISCUSSION TITLE
how can channels load 10million connections?

### DISCUSSION THREAD

**Discussion by jysatuo** [django/channels#1796] [Q&A]
hello,

I am learning to use channels,I have a big question,like the title,how can i load 10million connections by using channels?

I am in trouble, can you give me a good solution.

thank you very much.

**Comment by DarkcoderSe** [comment]
have you find answers?

**Comment by jysatuo** [comment]
lol,not yet. If you have answers just let me know.



At 2023-06-08 17:05:17, "Dark Coder" ***@***.***> wrote:

have you find answers?
